unstructured/partition/pdf.py

```diff
@@ -32,6 +33,8 @@
 from unstructured.partition.text import element_from_text, partition_text
 from unstructured.utils import requires_dependencies
 
+RE_MULTISPACE = re.compile(r"\s+")
```

Comment:
it looks like this is only used in the context of multiline replace below (line 236).

so to make the intent more clear, update to:

```
    RE_MULTISPACE_INCLUDING_NEWLINES = re.compile(r"\s+", re.DOTALL)
```
and use that instead?